### ISSUE TITLE
Improve import result message interface

### ISSUE THREAD

**Issue by rakleed** [ActivityWatch/activitywatch#394]
## Feature requests on the [forum](https://forum.activitywatch.net/t/message-internal-server-error-when-i-try-import-buckets/507/4)

**Is your feature request related to a problem? Please describe.**
Upon successful import, a `null` message is displayed, which is very difficult to associate with a successful import. And if import fails, the message `{"message": "Internal Server Error"}` is displayed all the time, which also does not say what the problem is.

**Describe the solution you'd like**
If import is successful, the message `Import completed successfully!` is displayed. And if import fails, a message is displayed that the import could not be completed, and the specific reason why (and also its solution is possible).

**Describe alternatives you've considered**

**Additional context**
The server should give the correct error messages, which can be displayed in the web interface to the user.

**Comment by ErikBjare** [comment]
Good catch, been thinking about this before but never opened an issue. We'll implement this in aw-server-rust, which is destined to become the default server soon™.

**Comment by johan-bjareholt** [comment]
@ErikBjare The datastore actually already returns the exact error in aw-server-rust, it's just a matter of prettifying it in the HTTP body response.

**Comment by nicolae-stroncea** [comment]
This would be slightly more complex (for a minor issue), but it would be great to stay on the same page, and instead just get the HTTP response in a box/alert that you can then close

**Comment by johan-bjareholt** [comment]
> This would be slightly more complex (for a minor issue), but it would be great to stay on the same page, and instead just get the HTTP response in a box/alert that you can then close

This is now in progress here:

https://github.com/ActivityWatch/aw-webui/pull/210

Next step is to return the actual error from aw-server and make the alert show that message, then we can consider this issue to be fixed.

**Comment by Rakleed** [comment]
.

**Comment by rakleed** [comment]
.

**Comment by rakleed** [comment]
.

**Comment by rakleed** [comment]
.

**Comment by rakleed** [comment]
ping

**Comment by rakleed** [comment]
Ping
